### ISSUE TITLE
Panic button when playing a midi file

### ISSUE THREAD

**Issue by trebmuh** [rncbc/qsynth#4]
When playing a MIDI file (drag & dropped, thanks ! ), I would expect the sound to shut down if I click on the "Panic" button. It doesn't happen here on version 0.4.1 .

**Comment by rncbc** [comment]
the Panic button **does** shut down the sound of playing MIDI file momentarily. however, it does **not** shutdown the MIDI player: the file continues playing to the end.

**Comment by trebmuh** [comment]
Yes that's right, I can confirm that. However, from a user perspective, one would expect a click to the "Panic" button to stop the playback of the MIDI file. Especially since the tooltip when mouse-overing the Panic button says "System reset". Would that be possible to implement?

**Comment by trebmuh** [comment]
@rncbc : ping

**Comment by rncbc** [comment]
@trebmuh : pong

as said,  the "Panic" button applies to the synth engine and not to libFluidSynth SMF player which qsynth uses as a client; IOW, none of the "Panic", "Reset" or "Restart" buttons affect the media transport of the embedded MIDI player.

**Comment by trebmuh** [comment]
Thanks for the detailed explanation.
